sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/ServiceBusTriggerAttribute.cs

```diff
@@ -115,7 +115,7 @@ public int MaxMessageBatchSize
         {
             get
             {
-                return _maxMessageBatchSize.HasValue ? _maxMessageBatchSize.Value : 1000;
```

Comment:
I think we can leave the default value as is and instead handle this in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Triggers/ServiceBusTriggerAttributeBindingProvider.cs#L140

Reply:
As discussed offline, the reason I have it updated here is as the worker extension is picking up this value (at least for the auto complete). As it was pointed by the customer in this https://github.com/Azure/azure-functions-dotnet-worker/issues/2142
So for now keeping it as is.